### DISCUSSION TITLE
Options for countdown & samples match playback rate should not be enabled by default in editor

### DISCUSSION THREAD

**Discussion by tilda** [ppy/osu#30822] [Ideas] [answered]
[Speaking from recent experience](https://osu.ppy.sh/beatmapsets/2271860/discussion/-/generalAll#/4689304) both of these options being enabled by default have seemed to cause a lot of confusion. In particular, samples match playback rate is not used often when mapping in stable as the option is fairly hidden behind multiple tabs and iirc only really used for keysounding stuff.

![image](https://github.com/user-attachments/assets/c0aad057-4140-43c6-9ede-735700d10b10)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/30826
